Console/Command/CleanUpAttributesAndValuesWithoutParentCommand.php

```diff
@@ -42,19 +42,27 @@ protected function configure()
         $this
             ->setName('eav:clean:attributes-and-values-without-parent')
             ->setDescription($description)
-            ->addOption('dry-run');
+            ->addOption('dry-run')
+            ->addOption('force');
     }
 
     public function execute(InputInterface $input, OutputInterface $output)
```

Comment:
We could add `int` as a return type now.

Reply:
Thanks for catching this. I've updated this now as intended initially.